setup.py

```diff
@@ -36,7 +36,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='2.9.16',
+    version='2.11.0',
```

Comment:
```suggestion
    version='2.10.0',
```

You also need to update `package.json`.

Reply:
2.10 is already in use (See setup.py in the master branch), hence 2.11